x/bank/internal/keeper/keeper_test.go

```diff
@@ -245,7 +246,42 @@ func TestVestingAccountSend(t *testing.T) {
 	// require that all vested coins are spendable plus any received
 	ctx = ctx.WithBlockTime(now.Add(12 * time.Hour))
 	err = app.BankKeeper.SendCoins(ctx, addr1, addr2, sendCoins)
-	vacc = app.AccountKeeper.GetAccount(ctx, addr1).(*auth.ContinuousVestingAccount)
+	vacc = app.AccountKeeper.GetAccount(ctx, addr1).(*vesting.ContinuousVestingAccount)
+	require.NoError(t, err)
+	require.Equal(t, origCoins, vacc.GetCoins())
+}
+
+func TestPeriodicVestingAccountSend(t *testing.T) {
```

Comment:
Ideally, the vesting account send/receive tests should be moved to `vesting`. Not sure if it introduces dep cycles

Reply:
I took a look at this, it does introduce cycles. `Vesting` would need access to `bank/internal/types` in order to avoid cycles. The current situation results in:

```
package github.com/cosmos/cosmos-sdk/x/auth/vesting/types
        imports github.com/cosmos/cosmos-sdk/x/bank
        imports github.com/cosmos/cosmos-sdk/x/bank/client/cli
        imports github.com/cosmos/cosmos-sdk/x/auth
        imports github.com/cosmos/cosmos-sdk/x/auth/simulation
        imports github.com/cosmos/cosmos-sdk/x/auth/vesting/types
```

Reply:
I see... let's fix this later then

Reply:
Why should these move to auth/vesting? These tests are testing banking functionality with relation to vesting contraints.